src/viam/sdk/services/mlmodel/mlmodel.cpp

```diff
@@ -60,23 +60,32 @@ MLModelService::tensor_info::string_to_data_type(const std::string& str) {
     // doesn't actually specify them.
     if (str == "int8") {
         return data_types::k_int8;
-    } else if (str == "uint8") {
+    }
+    if (str == "uint8") {
```

Comment:
Another silly one. New clang-tidy doesn't like `else` after `return` so I had to remove all the `else if`. OK i guess.

Reply:
thanks for adding all the `const`s and other "make `clang-tidy` happy" changes!

Reply:
I actually used `--fix` to automate applying most of them so it was pretty easy!

Reply:
Ahh, I figured there must have been some trick but wasn't sure what it was. Glad to know that `--fix` exists!